src/main/java/com/google/codeu/data/Message.java

```diff
@@ -56,4 +69,8 @@ public String getText() {
   public long getTimestamp() {
     return timestamp;
   }
+
+  public String getRecipient(){
+  return recipient;
+}
```

Comment:
The alignment of the last 2 lines is off a little.

Reply:
Oops - I think Github is saying that since I'm the author of this pull request I can't make the change.